src/compas/files/base_reader.py

```diff
@@ -129,8 +139,8 @@ def iter_lines(self):
             for line in fo:
                 yield line.rstrip()
 
-    def iter_chunks(self, chunk_size=1024):
-        """Yields lines from local binary files
+    def iter_chunks(self, chunk_size=4096):
```

Comment:
does the chunk size in any way depend on the type of binary encoding?

Reply:
The standard value should probably be removed.

I'm not sure of how this should work, I'm not experienced in reading binary files. But what I was going for was that the file reader specifies a reasonable chunk size and then reads amount of bytes corresponding to format spec.

Reply:
fair enough

Reply:
Do you have a chunk size you would like to have there instead or just change it to an arg?

Reply:
no leave it as it is